src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -180,10 +184,52 @@ MulticopterAttitudeControl::generate_attitude_setpoint(const Quatf &q, float dt,
 	// Align the desired tilt with the yaw setpoint
 	Quatf q_sp = q_sp_yaw * q_sp_rp;
 
-	q_sp.copyTo(attitude_setpoint.q_d);
-
 	// Transform to euler angles for logging only
+	Eulerf euler_des(q_sp);
+
+	/*** CUSTOM ***/
+
+	tilting_servo_sp_s servo_sp;
+
+	if(_param_airframe.get() == 12 ){ //If tilting_multirotors
+
+		//To do: change the time_constant of the filter from 0.0f to a param
+		_man_Fx_input_filter.setParameters(dt, 0.0f);
```

Comment:
Why is there a filter with time constant 0? Is it necessary? Did you tune that in your testing?

Reply:
This is not something that I've changed, it should be and old state of the code. I will change that according to the actual main branch